packages/wrangler/src/autoconfig/frameworks/vike.ts

```diff
@@ -0,0 +1,200 @@
+import assert from "node:assert";
+import { existsSync } from "node:fs";
+import path from "node:path";
+import { brandColor } from "@cloudflare/cli/colors";
+import * as recast from "recast";
+import { transformFile } from "../c3-vendor/codemod";
+import { installPackages } from "../c3-vendor/packages";
+import { isPackageInstalled } from "./utils/packages";
+import { Framework } from ".";
+import type { ConfigurationOptions, ConfigurationResults } from ".";
+import type { types } from "recast";
+
+const b = recast.types.builders;
+const t = recast.types.namedTypes;
+
+export class Vike extends Framework {
+	async configure({
+		projectPath,
+		dryRun,
+	}: ConfigurationOptions): Promise<ConfigurationResults> {
+		const vikeServerIsInstalled = isPackageInstalled(
+			"vike-server",
+			projectPath
+		);
+		if (vikeServerIsInstalled) {
+			// Note: if vike-server is used we just error, this is the simplest solution, alternatively we could
+			//       migrate the project as described in https://vike.dev/migration/vike-photon, but that is probably
+			//       not currently worth the effort (but something we might consider if/when needed)
+			throw new Error(
+				'Aborting since the project is using the deprecated "vike-server" package, please remove the package from your project and try again.'
+			);
+		}
+
+		if (!dryRun) {
+			// note: the following installation steps follow the guide in: https://vike.dev/cloudflare#get-started
+
+			await installPackages(["vike-photon", "@photonjs/cloudflare"]);
+			await installPackages(["@cloudflare/vite-plugin"], {
+				dev: true,
+				startText: "Installing vike-photon and @photonjs/cloudflare",
```

Comment:
This message refers to the previous installed packages

Reply:
good catch! 😓👍